Templates/VSIX/Project Templates/cpp-winui-template/WinUI3TemplateCpp/Scenario1_ShortName.xaml.cpp

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "Scenario1_ShortName.xaml.h"
+#if __has_include("Scenario1_ShortName.g.cpp")
+#include "Scenario1_ShortName.g.cpp"
+#endif
+
+using namespace winrt;
+using namespace Microsoft::UI::Xaml;
+using namespace Microsoft::UI::Xaml::Controls;
+using namespace Windows::Foundation;
+
+
+namespace winrt::$safeprojectname$::implementation
+{
+
+    MainPage Scenario1_ShortName::rootPage{ nullptr };
+
+    Scenario1_ShortName::Scenario1_ShortName()
+    {
+        InitializeComponent();
+        Scenario1_ShortName::rootPage = MainPage::Current();
+    }
+
+    void Scenario1_ShortName::NotifyUser(hstring const& strMessage, InfoBarSeverity const& severity)
```

Comment:
In the Windows samples, the "notify user" machinery goes on rootPage - so you say `rootPage.NotifyUser(...)` as the info bar belongs to the "containing" page of the sample - https://github.com/microsoft/Windows-universal-samples/blob/29d29659ac2821e5718024208ac1411535ff0785/Samples/AppServices/cppwinrt/AppServicesClient/KeepConnectionOpenScenario.cpp#L47-L50 has an example of this.